### ISSUE TITLE
Flags passed to nvcc for host configuration don't honor cfg

### ISSUE THREAD

**Issue by jsharpe** [bazel-contrib/rules_cuda#63]
The flags passed to the host compilation portion of nvcc don't honor the bazel configured toolchain flags.

Notably `-g` doesn't get propagated to the flags passed to nvcc when building with `-c dbg` and as a result no symbols are found.

**Comment by cloudhan** [comment]
Maybe a search path issue? For me, I need to use `dir bazel-rules_cuda` for symbol to work 
![image](https://user-images.githubusercontent.com/1279292/210468905-edb578f0-a4ac-4ab2-ac5a-1eee91c72b7c.png)

**Comment by jsharpe** [comment]
No, it's definitely missing the symbols, I have to add -g to host_copts to get it to work. This is specifically with the grailbio llvm 14 compiler as the host compiler so it may be down to some interaction with those rules.

**Comment by cloudhan** [comment]
Can you provide a repro? I might dig into it once I have time.

**Comment by jsharpe** [comment]
I'll have to try to find some time to isolate a reproducer, its within a reasonably complex build setup with pybind extensions so it's not easy to pull out straight away
